tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/SpecGenSdkConfigHelper.cs

```diff
@@ -15,7 +15,8 @@ public interface ISpecGenSdkConfigHelper
     {
         // Config value retrieval methods
         Task<T> GetConfigValueFromRepoAsync<T>(string repositoryRoot, string jsonPath);
-        Task<(BuildConfigType type, string value)> GetBuildConfigurationAsync(string repositoryRoot);
+        Task<(ConfigContentType type, string value)> GetBuildConfigurationAsync(string repositoryRoot);
```

Comment:
Why can't we use just GetConfigurationAsync from caller instead of adding a wrapper for get build config which is just calling GetConfigurationAsync?